kernel-open/common/inc/nv_uvm_interface.h

```diff
@@ -194,7 +194,7 @@ void nvUvmInterfaceAddressSpaceDestroy(uvmGpuAddressSpaceHandle vaSpace);
     nvUvmInterfaceMemoryAllocFB
 
     This function will allocate video memory and provide a mapped Gpu
-    virtual address to this allocation. It also returns the Gpu physical
+    virtual address to this allocation. It also returns the Gpu physical 
```

Comment:
Not sure why this is needed when a new line already exists? :smiley:

Reply:
It is not a new line. It's just a space that I accidentally added. Thanks for pointing it out!

Reply:
Removed in 298b692.